PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,17 @@
 <!--
 Thanks for your contribution!
 
-Reviewing pull requests takes a lot of time and we're all volunteers. Please make sure you go through the following checklist and all items before pinging someone for a review.
+Reviewing pull requests takes a lot of time and we're all volunteers. Please make sure you go through the following checklist and all items.
 -->
 
+**For short term contributors:** we understand that getting your commits sharp and defined like we require it is a hard tasks and takes some learning. If you look to help us out without wanting to contribute long term there's no need for you to learn this. Just drop us a message and we'll take care of brushing up your stuff for merge!
+
 ### Checklist
 
-- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
-- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
-- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
-- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
-- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
-- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!
+- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed them.
+- [ ] I ran coala over my code locally. (*All commits have to pass individually.* It is not sufficient to have "fixup commits" on your PR, our bot will still report the issues for the previous commit.) You will likely receive a lot of bot comments and build failures if coala does not pass on every single commit!
```

Comment:
This feels like to much text (also linebreaks?)
Maybe just add the testing docs link for reference